rtengine/histmatching.cc

```diff
@@ -86,12 +86,19 @@ void mappingToCurve(const std::vector<int> &mapping, std::vector<double> &curve)
     curve.clear();
     
     const int npoints = 8;
-    int idx = 1;
+    int idx = 15;
     for (; idx < int(mapping.size()); ++idx) {
         if (mapping[idx] >= idx) {
             break;
         }
     }
+    if (idx == int(mapping.size())) {
+        for (idx = 1; idx < int(mapping.size()); ++idx) {
+            if (mapping[idx] >= idx) {
+                break;
+            }
+        }
+    }
     int step = max(int(mapping.size())/npoints, 1);
```

Comment:
Which `max()` is that?

Reply:
max Von sydow -- I thought it was obvious ;-P

or are you saying we have different implementaions of max that give different results *over the integers*?

Reply:
Aha, I thought it was `max_planck()`. 😄

> or are you saying we have different implementaions of max that give different results over the integers?

No, we have `std::max()` and `rtengine::max()` both yielding the same result, with the latter even being `constexpr` and taking more than two arguments. But without giving the namespace it's unclear which one is used. I bit further below you use `std::max()`, so it would be more consistent to add the namespace here, too (and do without `using namespace` for these very reasons).